app/models/nic/interface.rb

```diff
@@ -20,10 +20,28 @@ class Interface < Base
 
     validate :alias_subnet
 
-    delegate :network, :to => :subnet
+    delegate :network, :to => :subnet, :prefix => true
+    delegate :network, :to => :subnet6, :prefix => true
```

Comment:
`network6`? I'm not quite sure how this works if two delegates with the same name point to 2 different associations